package.json

```diff
@@ -164,16 +157,18 @@
     "singleQuote": true,
     "trailingComma": "none"
   },
-  "jest": {
-    "testEnvironment": "node",
-    "coverageThreshold": {
-      "global": {
-        "statements": 100
-      }
-    }
-  },
   "clean-publish": {
     "cleanDocs": true,
     "cleanComments": true
+  },
+  "c8": {
```

Comment:
Here is my universal config https://github.com/ai/boilerplates/blob/main/esm-lib/package.json#L40-L46